doc/sphinx/source/releasenotes.md

```diff
@@ -7,6 +7,14 @@ for each release of libCEED.
 
 ## Current `main` branch
 
+### Interface changes
+
+- QFunction and Operator field objects, `CeedQFunctionField` and `CeedOperatorField`, and associated getters, {c:func}`CeedQFunctionGetFields`; {c:func}`CeedQFunctionFieldGetName`; {c:func}`CeedQFunctionFieldGetSize`; {c:func}`CeedQFunctionFieldGetEvalMode`; {c:func}`CeedOperatorGetFields`; {c:func}`CeedOperatorFieldGetElemRestriction`; {c:func}`CeedOperatorFieldGetBasis`; and {c:func}`CeedOperatorFieldGetVector`, promoted to the public API.
+
+### New features
+
+- `CeedScalar` can now be set as `float` or `double` at compile time.
```

Comment:
Not sure what more to say. That was a big PR, but this seems to cover the key point.

Reply:
I think this is fine at this time -- we have more ambitious plans longer term.